src/pages/sign-on/components/desktop/ForgotPasswordHelper.module.css

```diff
@@ -0,0 +1,49 @@
+.questionButtonIcon {
+  margin-right: 10px;
+}
+
+.modalWidth {
+  width: 510px;
+}
+
+.modalBody {
+  margin: 24px;
+}
+
+.restoreAccessText {
+  text-align: center;
+  font-size: var(--font-m);
+  line-height: 24px;
+  font-weight: bold;
+  margin-bottom: 24px;
+}
+
+.emailContainer {
+  display: flex;
+  flex-direction: row;
+  padding: 12px 24px;
+  border: 1px solid #DAD9E0;
```

Comment:
Not your problem to fix now, but we should pull #DAD9E0 into a var. It's clearly used a few places in the app :)